lib/utils.dart

```diff
@@ -985,7 +1068,8 @@ Future<List<Map<String, dynamic>>> dumpWmicQuery(String section) async {
 
       try {
         value = num.parse(value);
-      } catch (e) {}
+      } catch (e) {
```

Comment:
Clarification: This isn't caught because it's just turning it into a number if it parses as a number